hack/update-codecgen.sh

```diff
@@ -117,7 +117,7 @@ fi
 
 echo "Building codecgen"
 CODECGEN="${PWD}/codecgen_binary"
-godep go build -o "${CODECGEN}" github.com/ugorji/go/codec/codecgen
+go build -o "${CODECGEN}" ./vendor/github.com/ugorji/go/codec/codecgen
```

Comment:
something about the leading `./` seems to bother me, but I can't put my finger on it. Probably sohuld just be quiet.